### DISCUSSION TITLE
Is it possible to specify patch version when creating patches

### DISCUSSION THREAD

**Discussion by mohkale** [magit/magit#4928] [Support] [answered]
From what I understand when creating a second iteration of patches you're supposed to supply -v2 (or whatevcer version you're using) to git-send-email or git-format-patches. Is this possible with magit?

**Comment by kyleam** [comment, marked as answer]
Yes, the `magit-patch-create` transient includes `--reroll-count`:

https://github.com/magit/magit/blob/f59df798194d4213b5a2b2f43a2ee25b71cbbb90/lisp/magit-patch.el#L78-L80

https://github.com/magit/magit/blob/f59df798194d4213b5a2b2f43a2ee25b71cbbb90/lisp/magit-patch.el#L157-L163

**Reply by mohkale** [reply]
Ah, my bad, I've only ever used the short form -v option (assumed it was short for --version :rofl:). This works. Thanks kyleam.
